### PR TITLE
Migrate the Expense Related Stories to Storybook

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/4313

### CODE REVIEW

**Comment by SudharakaP** [comment]
@Betree : All expense related stories have been migrated to Storybook and seems to work fine. One point though is that you may notice that I've commented out `ExpenseForm.stories.mdx`;

https://github.com/opencollective/opencollective-frontend/blob/76c38f7eba7389c24094305399b8e68c94b96823/stories/expenses/ExpenseForm.stories.mdx#L1

This is due to it being using the `I18nAddressFields.js`. There seems to be some problem with serialization which Storybook doesn't seem to handle and I couldn't find a workaround for this for the moment; the relevant (or similar) issue is found at, https://github.com/storybookjs/storybook/issues/15181. 🤔 🐼

**Comment by SudharakaP** [comment]
And next weekend I think we can finish off the rest of the stuff and hopefully deploy this soon. 😄 😄 😄

File: stories/expenses/ExpensesFilters.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import ExpensesFilters from '../../components/expenses/ExpensesFilters';
+import { webpackCollective } from '../mocks/collectives';
+
+<Meta title="Expenses/ExpensesFilters" component={ExpensesFilters} />
+
+<Canvas>
+  <Story name="Receipt type (file required)">
```
Comment (Betree) [inline stories/expenses/ExpensesFilters.stories.mdx:8]:
```suggestion
  <Story name="Default">
```

**Review by Betree — APPROVED** [review]
Woohoo this is great! We should start considering moving https://styleguide.opencollective.com to this new styleguide.

**Review by snitin315 — APPROVED** [review]
Nice.

**Pull request merged**